addons/mail/static/src/discuss/call/common/ptt_unreliable_banner.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="discuss.pttUnreliableBanner">
+    <span t-if="isVisible" class="o-discuss-PttUnreliableBanner alert alert-info px-2 py-1 m-0 rounded-0">
+        <div class="d-flex justify-content-between align-items-center">
+            <span class="small">
+                    The Push-to-Talk feature is only accessible within tab focus.
+                    <t t-if="pttExtService.isAvailable">
```

Comment:
I would suggest to always display it, users can decide to use a browser or to install it or not.